_posts/0000-01-04-rajeshmasuti.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![steve-harvey](https://cloud.githubusercontent.com/assets/16547949/25401187/0af0e288-29c3-11e7-9a88-38efd8b375cc.jpg)
```

Comment:
I don't really think so

Reply:
Steve has a habit of making the wrong announcement. :smiley: